foo/lib/config/configloader.py

```diff
@@ -0,0 +1,98 @@
+import os
```

Comment:
Could you tell me why the folder structure is so nested?
Also consider simplifying the filename e.g. `config.py`